core/sys/rtimer.h

```diff
@@ -57,9 +57,11 @@
 
 #ifndef RTIMER_CLOCK_LT
```

Comment:
`RTIMER_CLOCK_LT` needs to be changed to `RTIMER_CLOCK_DIFF`, there and in the comments in the other files.